analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/RequestsWithExcessiveLength.vb

```diff
@@ -78,6 +78,19 @@ Namespace Tests.TestCases
             Return Nothing
         End Function
 
+        <HttpPost>
+        <RequestSizeLimit(Integer.MaxValue)> ' Noncompliant
+        Public Function RequestSizeLimitWithoutNumericLiteral() As ActionResult
+            Return Nothing
+        End Function
+
+        <HttpPost>
+        <RequestFormLimits(MultipartBodyLengthLimit:=1000000000)> ' Noncompliant [1]
```

Comment:
what happens when
- `RequestFormLimits MultipartBodyLengthLimit` is above limit and `RequestSizeLimit` is below limit - will the secondary location appear as well?
- viceversa: `RequestSizeLimit` is above limit and ``RequestFormLimits MultipartBodyLengthLimit`` is above limit?